example/src/test/java/com/capgemini/mythaistar/features/bookTable/bookTableScenario.feature

```diff
@@ -0,0 +1,55 @@
+@Test
+Feature: My Thai Star book table page
+
+  Background:
+    Given I am on the book table page
+    And I click on Date and time field
+
+  Scenario Outline: Test book a table form with different data
+    And I set <date> date and <hour> hour and <minutes> minutes
+    When I complete action
+    And I set name field with <name> value
+    And I set email field with <email> value
+    And I set table guests field with <guests> value
+    And I check accept terms checkbox
+    When I click book table button
+    Then I should see <status> of reservation
```

Comment:
should be: I should see <status> status of reservation on X page; X replace with proper page name